tree/ntuple/v7/src/RPageStorageDaos.cxx

```diff
@@ -537,11 +546,17 @@ void ROOT::Experimental::Detail::RPageSourceDaos::LoadSealedPage(DescriptorId_t
       pageInfo = clusterDescriptor.GetPageRange(columnId).Find(clusterIndex.GetIndex());
    }
 
+   if (pageInfo.fLocator.fReserved & Internal::EDaosLocatorFlags::kCagedPage) {
+         throw ROOT::Experimental::RException(
+            R__FAIL("LoadSealedPage: accessing caged pages is only supported in conjunction with cluster cache"));
```

Comment:
Perhaps we could remove the `LoadSealedPage: ` prefix, as that can be determined from the exception itself (same in [line 584](https://github.com/root-project/root/pull/11466/files/4843557bf08aa0da2ccaaaf459be9c1ca8bac730..cdb5f86aa3a18178b3be6ddba06d9bac56bc6b63#diff-b6cfded99bfdceda6f816193a6fadac72cd4b0a606460f773aa5e1f058f4f643R584)).